app/views/research_sessions/preview.html.erb

```diff
@@ -76,13 +76,13 @@
   <section>
     <h3 class="subtitle-small" id="doi">Do I have to take part?</h3>
     <p>
-      Taking part is entirely voluntary - it is up to you to decide whether or not
-      to take part. If you decide to take part you do not have to answer questions
-      you do not want to answer. You can also change your mind about taking part
+      Taking part is entirely voluntary - it is up to you to decide whether or not <%= @research_session.able_to_consent? ? 'you' : 'your child/the child in your care' %>
+      to take part. If you decide <%= @research_session.able_to_consent? ? 'you' : 'your child/the child in your care' %> to take part <%= @research_session.able_to_consent? ? 'you' : 'they' %> do not have to answer questions
```

Comment:
This will result in some weird English, e.g. "If you decide you to take part" or "If you decide your child/the child in your care to take part"

Reply:
Yeah was taking me a while to figure this one out.